modules/mobile/app/helpers/mobile/appointments_cache_interface.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
```

Comment:
The hardest part of this ticket was trying to figure out where this service object should go and what it should be called. Currently, we have three areas where we're putting various service objects:
- helpers: in a typical rails app, this would refer to view helpers, but since we don't have view helpers the namespace was open. There are a few items in this folder and they're mostly just what I would call service objects (i.e., a ruby object that handles some isolated logic).
- services: this is where I would normally put a file like this. However, it appears to me that our services folder was originally intended for making external calls. Most of the classes here sub-class Common::Client::Base and often have an accompanying configuration and sometimes a proxy that handles the logic for interfacing with data coming from upstream. All of the services in this directory are also divided into v0 and v2 sub-directories, which further makes it seem like they're intended to be directly linked to upstream services. Given that this class uses both v0 and v2 services, it didn't really belong in either folder. 
- lib: this contains a vaos appointments helper that I would also consider a service object. IMO this was probably added here by mistake. Lib is typically for code that is not specific to the rails app and could be packaged as a gem. Lib is not in the autoload path, so any classes here must be manually required, which recently caused a bug when some code that worked in specs failed on staging. 

So I've added this to helpers for now, but I would love for us to have a group conversation about how we want our app organized. IMO, it might make the most sense for the service folder to be reserved for subclasses of Common::Client::Base and we can put our logic in helpers or elsewhere.

Reply:
My vote is for helpers. I definitely think it deviates from norms in the rails community and in another project I'd advocate for other locations, but given the current layout of the directories, helpers seems like the best spot imo